src/dynamicprompts/parser/config.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class ParserConfig:
+    left_brace: str = "{"
+    right_brace: str = "}"
```

Comment:
I wonder if these should be named `variant_start` and `variant_end` for future expansion..?